src/main/webapp/js/instructor.js

```diff
@@ -152,6 +152,51 @@ function setupFsCopyModal() {
             }
         });
     });
+    
+    $('#form_copy_list').submit(fsCopyModalFormSubmitHandler);
+}
+
+/**
+ * Event handler for the copying feedback session modal's form.
+ * For submitting the form by ajax.
+ * 
+ * this should be bound to the form element
+ */
+function fsCopyModalFormSubmitHandler(e) {
```

Comment:
Should this be an anonymous function directly in the submit call above?
As in: `$___.submit(function (e) { ... })`
That ensures it will be bound to the element and won't be called from anywhere else.